<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeWidget</class>
 <widget class="QWidget" name="RangeWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RangeWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="RangeWidgetGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable/Disable cluster range (when &lt;span style=&quot; font-weight:600;&quot;&gt;Disabled&lt;/span&gt; - selected all range)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_to">
        <property name="text">
         <string>To</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="rangeEnd"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="rangeBegin"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_from">
        <property name="text">
         <string>From</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>RangeWidgetGroup</tabstop>
  <tabstop>rangeBegin</tabstop>
  <tabstop>rangeEnd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
